TestingTravis/ViewController.swift

```diff
@@ -13,6 +13,8 @@ class ViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view, typically from a nib.
+        
+        //testing PR for CI ddd
     }
 
 
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Nakul97/TestingTravis/blob/2ab967952d0734b15797e4d70bbe1b3c3fb02e82/TestingTravis/ViewController.swift#L20"></span>Limit vertical whitespace to a single empty line. Currently 2.<br />
<code>vertical_whitespace</code> <code>ViewController.swift:20</code></td>
    </tr>
  </tbody>
</table>